test/test_jit_cuda_fuser.py

```diff
@@ -331,14 +333,14 @@ def where(x: torch.Tensor, y: torch.Tensor, cond: torch.Tensor):
         where_jit = torch.jit.script(where)
         self._run_helper(where_jit, where, x, y, cond)
 
-        def lerp(x : torch.Tensor, y : torch.Tensor, z : torch.Tensor):
+        def lerp(x: torch.Tensor, y: torch.Tensor, z: torch.Tensor):
```

Comment:
When you specify the types for inputs and outputs in python for torch.script, I thought you also had to specify the output type.  Is that not true?

Reply:
You could but I thought that's optional (some of their examples definitely don't have it.)
https://pytorch.org/docs/stable/jit_language_reference.html#variable-resolution